src/test/resources/regressions/features/sequences/seq-contains-simple1.gobra

```diff
@@ -38,11 +38,11 @@ func example6(x int, ghost xs seq[int], ghost ys seq[bool]) {
 }
 
 requires 0 <= n
-ensures !(n in seq[0..n])
```

Comment:
seems to occur in many files. I'm fine with having no spaces in some files such that both ways are tested. But in the interest of keeping the diff as small as possible, I would revert it in most files